concent_api/core/tests/test_payments_bankster.py

```diff
@@ -411,38 +401,52 @@ def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount
 
         self.assertEqual(claim_against_requestor.amount, subtask_cost)
 
-    def test_that_settle_overdue_acceptances_should_raise_exception_if_any_transaction_from_acceptances_list_is_before_youngest_transaction_timestamp(self):
+    def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount_paid_when_requesting_payment_for_many_subtasks(self):
```

Comment:
Those are not `many_subtasks`, but exactly two.

Reply:
True xD can I change it at least to "multiple"?